### PR TITLE
Mul 217 hide primary order statistics

### PR DESCRIPTION
**Pull request by MacBalc**

### CODE REVIEW

File: features/admin/payment_viewing.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    And the store operates on a channel named "Web-EU" in "EUR" currency and with hostname "web-eu"
+    And that channel allows to shop using "EUR" and "GBP" currencies
+    And the store has country "Ireland"
+    And the store has a product "Leprechaun's Gold" priced at "€10.00" in "Web-EU" channel
+    And the store has a zone "EU"
+    And the store has customer "example@user.com"
+    And the store has "UPS" shipping method with "$20.00" fee per unit for "Web-EU" channel
+    And the store has also a payment method "Bank transfer" with a code "transfer"
+
+
```
Comment (PiotrSzymanski2000) [inline features/admin/payment_viewing.feature:18]: No blank line needed

File: features/admin/shipment_viewing.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    And the store operates on a channel named "Web-EU" in "EUR" currency and with hostname "web-eu"
+    And that channel allows to shop using "EUR" and "GBP" currencies
+    And the store has country "Ireland"
+    And the store has a product "Leprechaun's Gold" priced at "€10.00" in "Web-EU" channel
+    And the store has a zone "EU"
+    And the store has customer "example@user.com"
+    And the store has "UPS" shipping method with "$20.00" fee per unit for "Web-EU" channel
+    And the store has also a payment method "Bank transfer" with a code "transfer"
+
+
```
Comment (PiotrSzymanski2000) [inline features/admin/shipment_viewing.feature:18]: No blank line needed

File: features/shop/order.feature [hunk 3]
```diff
@@ -13,36 +13,32 @@ Feature: Spliting orders when cart was filled with products from different Vendo
...
     Then I should see 1 orders
+
+  @ui
+  Scenario: Do not Assign number to primary order
+    Given store has 4 products from different Vendors
+    And I have 3 products in cart
+    And I finalize order
+    Then primary order should not have number
+
+
```
Comment (PiotrSzymanski2000) [inline features/shop/order.feature:35]: No blank line needed

**Review by PiotrSzymanski2000 — APPROVED** [review]

**Comment by senghe** [comment]
I'm closing this, as it'll be merged with #83

**Pull request closed without merging**